### ISSUE TITLE
communication/azure-communication-phonenumbers - cspell found spelling errors in source code

### ISSUE THREAD

**Issue by danieljurek** [Azure/azure-sdk-for-python#22639]
Spell check scanning of package at `sdk/communication/azure-communication-phonenumbers` detected spelling errors in the source code and related files. This directory is opted out of PR spell checking in PR #22199 to keep PRs unblocked.

## What to do

1. Ensure Node.js is installed (https://nodejs.org/en/download/).
1. Delete the entry in `.vscode/cspell.json`'s `ignorePaths` field. It will look like: `sdk/communication/azure-communication-phonenumbers/**`. You need to do this to enable checking the files.
1. From the root of the repo run spell check using `./eng/common/spelling/Invoke-Cspell.ps1 -ScanGlobs "sdk/communication/azure-communication-phonenumbers/**"`
1. Fix detections according to http://aka.ms/azsdk/engsys/spellcheck use the "False positives" section to fix false positives
1. Check in changes (including the change to `.vscode/cspell.json` where the `ignorePaths` is updated to remove the entry for this service.). You may need to run `git add -f .vscode/cspell.json` to force adding the changes to the file in git.

## Spell checking output

```
./sdk/communication/azure-communication-phonenumbers/azure/communication/phonenumbers/_shared/models.py:28:18 - Unknown word (enviornment)
./sdk/communication/azure-communication-phonenumbers/azure/communication/phonenumbers/_shared/models.py:32:5 - Unknown word (GCCH)
./sdk/communication/azure-communication-phonenumbers/azure/communication/phonenumbers/_shared/models.py:32:13 - Unknown word (GCCH)
./sdk/communication/azure-communication-phonenumbers/azure/communication/phonenumbers/_shared/policy.py:11:46 - Unknown word (IOHTTP)
./sdk/communication/azure-communication-phonenumbers/azure/communication/phonenumbers/_shared/policy.py:14:33 - Unknown word (IOHTTP)
./sdk/communication/azure-communication-phonenumbers/azure/communication/phonenumbers/_shared/user_credential_async.py:53:43 - Unknown word (inprogress)
./sdk/communication/azure-communication-phonenumbers/azure/communication/phonenumbers/_shared/user_credential_async.py:77:26 - Unknown word (inprogress)
./sdk/communication/azure-communication-phonenumbers/azure/communication/phonenumbers/_shared/user_credential.py:55:37 - Unknown word (inprogress)
./sdk/communication/azure-communication-phonenumbers/azure/communication/phonenumbers/_shared/user_credential.py:77:20 - Unknown word (inprogress)
./sdk/communication/azure-communication-phonenumbers/azure/communication/phonenumbers/_shared/utils.py:27:25 - Unknown word (timegm)
./sdk/communication/azure-communication-phonenumbers/CHANGELOG.md:5:89 - Unknown word (PSTN)
./sdk/communication/azure-communication-phonenumbers/CHANGELOG.md:6:82 - Unknown word (PSTN)
./sdk/communication/azure-communication-phonenumbers/CHANGELOG.md:9:24 - Unknown word (verion)
./sdk/communication/azure-communication-phonenumbers/samples/managed_identity_authentication_sample.py:18:78 - Unknown word (aplication)
./sdk/communication/azure-communication-phonenumbers/samples/update_phone_number_capabilities_sample.py:12:37 - Unknown word (updtae)
./sdk/communication/azure-communication-phonenumbers/test/recordings/test_phone_number_administration_client_async.test_get_purchased_phone_number_from_managed_identity.yaml:24:23 - Unknown word (NOCACHE)
./sdk/communication/azure-communication-phonenumbers/test/recordings/test_phone_number_administration_client_async.test_get_purchased_phone_number_with_invalid_phone_number.yaml:23:23 - Unknown word (EKUG)
./sdk/communication/azure-communication-phonenumbers/test/recordings/test_phone_number_administration_client_async.test_get_purchased_phone_number_with_invalid_phone_number.yaml:26:23 - Unknown word (NOCACHE)
./sdk/communication/azure-communication-phonenumbers/test/recordings/test_phone_number_administration_client_async.test_get_purchased_phone_number.yaml:28:23 - Unknown word (NOCACHE)
./sdk/communication/azure-communication-phonenumbers/test/recordings/test_phone_number_administration_client_async.test_list_purchased_phone_numbers_from_managed_identity.yaml:421:23 - Unknown word (NOCACHE)
./sdk/communication/azure-communication-phonenumbers/test/recordings/test_phone_number_administration_client_async.test_list_purchased_phone_numbers_from_managed_identity.yaml:847:23 - Unknown word (NOCACHE)
./sdk/communication/azure-communication-phonenumbers/test/recordings/test_phone_number_administration_client_async.test_list_purchased_phone_numbers_from_managed_identity.yaml:945:29 - Unknown word (trco)
./sdk/communication/azure-communication-phonenumbers/test/recordings/test_phone_number_administration_client_async.test_list_purchased_phone_numbers_from_managed_identity.yaml:948:23 - Unknown word (NOCACHE)
./sdk/communication/azure-communication-phonenumbers/test/recordings/test_phone_number_administration_client_async.test_list_purchased_phone_numbers.yaml:425:23 - Unknown word (NOCACHE)
./sdk/communication/azure-communication-phonenumbers/test/recordings/test_phone_number_administration_client_async.test_list_purchased_phone_numbers.yaml:852:26 - Unknown word (Ignf)
./sdk/communication/azure-communication-phonenumbers/test/recordings/test_phone_number_administration_client_async.test_list_purchased_phone_numbers.yaml:855:23 - Unknown word (NOCACHE)
./sdk/communication/azure-communication-phonenumbers/test/recordings/test_phone_number_administration_client_async.test_list_purchased_phone_numbers.yaml:960:23 - Unknown word (NOCACHE)
./sdk/communication/azure-communication-phonenumbers/test/recordings/test_phone_number_administration_client_async.test_purchase_phone_numbers_from_managed_identity.yaml:24:29 - Unknown word (Taydz)
./sdk/communication/azure-communication-phonenumbers/test/recordings/test_phone_number_administration_client_async.test_purchase_phone_numbers_from_managed_identity.yaml:76:20 - Unknown word (bhlk)
./sdk/communication/azure-communication-phonenumbers/test/recordings/test_phone_number_administration_client_async.test_purchase_phone_numbers_from_managed_identity.yaml:105:18 - Unknown word (Lpfpe)
./sdk/communication/azure-communication-phonenumbers/test/recordings/test_phone_number_administration_client_async.test_purchase_phone_numbers_from_managed_identity.yaml:105:32 - Unknown word (Ikew)
./sdk/communication/azure-communication-phonenumbers/test/recordings/test_phone_number_administration_client_async.test_purchase_phone_numbers.yaml:28:23 - Unknown word (Eydd)
./sdk/communication/azure-communication-phonenumbers/test/recordings/test_phone_number_administration_client_async.test_purchase_phone_numbers.yaml:28:32 - Unknown word (IRNA)
./sdk/communication/azure-communication-phonenumbers/test/recordings/test_phone_number_administration_client_async.test_purchase_phone_numbers.yaml:88:26 - Unknown word (EWWX)
./sdk/communication/azure-communication-phonenumbers/test/recordings/test_phone_number_administration_client_async.test_purchase_phone_numbers.yaml:121:27 - Unknown word (Ybsvom)
./sdk/communication/azure-communication-phonenumbers/test/recordings/test_phone_number_administration_client_async.test_purchase_phone_numbers.yaml:151:28 - Unknown word (NRFJ)
./sdk/communication/azure-communication-phonenumbers/test/recordings/test_phone_number_administration_client_async.test_purchase_phone_numbers.yaml:180:17 - Unknown word (hiqi)
./sdk/communication/azure-communication-phonenumbers/test/recordings/test_phone_number_administration_client_async.test_purchase_phone_numbers.yaml:180:27 - Unknown word (CPHV)
./sdk/communication/azure-communication-phonenumbers/test/recordings/test_phone_number_administration_client_async.test_search_available_phone_numbers_from_managed_identity.yaml:24:19 - Unknown word (lzjb)
./sdk/communication/azure-communication-phonenumbers/test/recordings/test_phone_number_administration_client_async.test_search_available_phone_numbers_from_managed_identity.yaml:24:23 - Unknown word (EWKCY)
./sdk/communication/azure-communication-phonenumbers/test/recordings/test_phone_number_administration_client_async.test_search_available_phone_numbers_from_managed_identity.yaml:29:23 - Unknown word (NOCACHE)
./sdk/communication/azure-communication-phonenumbers/test/recordings/test_phone_number_administration_client_async.test_search_available_phone_numbers_with_invalid_country_code.yaml:28:24 - Unknown word (Krgv)
./sdk/communication/azure-communication-phonenumbers/test/recordings/test_phone_number_administration_client_async.test_search_available_phone_numbers_with_invalid_country_code.yaml:28:28 - Unknown word (VICL)
./sdk/communication/azure-communication-phonenumbers/test/recordings/test_phone_number_administration_client_async.test_search_available_phone_numbers_with_invalid_country_code.yaml:31:23 - Unknown word (NOCACHE)
./sdk/communication/azure-communication-phonenumbers/test/recordings/test_phone_number_administration_client_async.test_search_available_phone_numbers.yaml:28:25 - Unknown word (Rbaw)
./sdk/communication/azure-communication-phonenumbers/test/recordings/test_phone_number_administration_client_async.test_search_available_phone_numbers.yaml:29:37 - Unknown word (ccca)
./sdk/communication/azure-communication-phonenumbers/test/recordings/test_phone_number_administration_client_async.test_search_available_phone_numbers.yaml:30:68 - Unknown word (ccca)
./sdk/communication/azure-communication-phonenumbers/test/recordings/test_phone_number_administration_client_async.test_search_available_phone_numbers.yaml:33:23 - Unknown word (NOCACHE)
./sdk/communication/azure-communication-phonenumbers/test/recordings/test_phone_number_administration_client_async.test_update_phone_number_capabilities_from_managed_identity.yaml:24:23 - Unknown word (NOCACHE)
./sdk/communication/azure-communication-phonenumbers/test/recordings/test_phone_number_administration_client_async.test_update_phone_number_capabilities_from_managed_identity.yaml:52:25 - Unknown word (BXAQ)
./sdk/communication/azure-communication-phonenumbers/test/recordings/test_phone_number_administration_client_async.test_update_phone_number_capabilities_from_managed_identity.yaml:53:43 - Unknown word (efed)
./sdk/communication/azure-communication-phonenumbers/test/recordings/test_phone_number_administration_client_async.test_update_phone_number_capabilities_from_managed_identity.yaml:54:74 - Unknown word (efed)
./sdk/communication/azure-communication-phonenumbers/test/recordings/test_phone_number_administration_client_async.test_update_phone_number_capabilities_from_managed_identity.yaml:57:23 - Unknown word (NOCACHE)
./sdk/communication/azure-communication-phonenumbers/test/recordings/test_phone_number_administration_client_async.test_update_phone_number_capabilities_from_managed_identity.yaml:81:16 - Unknown word (Ffqg)
./sdk/communication/azure-communication-phonenumbers/test/recordings/test_phone_number_administration_client_async.test_update_phone_number_capabilities_from_managed_identity.yaml:81:27 - Unknown word (xfddz)
./sdk/communication/azure-communication-phonenumbers/test/recordings/test_phone_number_administration_client_async.test_update_phone_number_capabilities_from_managed_identity.yaml:84:23 - Unknown word (NOCACHE)
./sdk/communication/azure-communication-phonenumbers/test/recordings/test_phone_number_administration_client_async.test_update_phone_number_capabilities_from_managed_identity.yaml:107:32 - Unknown word (ZOQQ)
./sdk/communication/azure-communication-phonenumbers/test/recordings/test_phone_number_administration_client_async.test_update_phone_number_capabilities_from_managed_identity.yaml:110:23 - Unknown word (NOCACHE)
./sdk/communication/azure-communication-phonenumbers/test/recordings/test_phone_number_administration_client_async.test_update_phone_number_capabilities_with_invalid_phone_number.yaml:27:30 - Unknown word (NAOQ)
./sdk/communication/azure-communication-phonenumbers/test/recordings/test_phone_number_administration_client_async.test_update_phone_number_capabilities_with_invalid_phone_number.yaml:30:23 - Unknown word (NOCACHE)
./sdk/communication/azure-communication-phonenumbers/test/recordings/test_phone_number_administration_client_async.test_update_phone_number_capabilities.yaml:25:21 - Unknown word (Ipkup)
./sdk/communication/azure-communication-phonenumbers/test/recordings/test_phone_number_administration_client_async.test_update_phone_number_capabilities.yaml:25:26 - Unknown word (IIFU)
./sdk/communication/azure-communication-phonenumbers/test/recordings/test_phone_number_administration_client_async.test_update_phone_number_capabilities.yaml:28:23 - Unknown word (NOCACHE)
./sdk/communication/azure-communication-phonenumbers/test/recordings/test_phone_number_administration_client_async.test_update_phone_number_capabilities.yaml:65:23 - Unknown word (NOCACHE)
./sdk/communication/azure-communication-phonenumbers/test/recordings/test_phone_number_administration_client_async.test_update_phone_number_capabilities.yaml:96:23 - Unknown word (NOCACHE)
./sdk/communication/azure-communication-phonenumbers/test/recordings/test_phone_number_administration_client_async.test_update_phone_number_capabilities.yaml:123:26 - Unknown word (mjph)
./sdk/communication/azure-communication-phonenumbers/test/recordings/test_phone_number_administration_client_async.test_update_phone_number_capabilities.yaml:126:23 - Unknown word (NOCACHE)
./sdk/communication/azure-communication-phonenumbers/test/recordings/test_phone_number_administration_client.test_get_purchased_phone_number_from_managed_identity.yaml:35:16 - Unknown word (NOCACHE)
./sdk/communication/azure-communication-phonenumbers/test/recordings/test_phone_number_administration_client.test_get_purchased_phone_number_with_invalid_phone_number.yaml:31:10 - Unknown word (Rnvsv)
./sdk/communication/azure-communication-phonenumbers/test/recordings/test_phone_number_administration_client.test_get_purchased_phone_number_with_invalid_phone_number.yaml:37:16 - Unknown word (NOCACHE)
./sdk/communication/azure-communication-phonenumbers/test/recordings/test_phone_number_administration_client.test_get_purchased_phone_number.yaml:33:13 - Unknown word (Uhan)
./sdk/communication/azure-communication-phonenumbers/test/recordings/test_phone_number_administration_client.test_get_purchased_phone_number.yaml:33:18 - Unknown word (Uuedv)
./sdk/communication/azure-communication-phonenumbers/test/recordings/test_phone_number_administration_client.test_get_purchased_phone_number.yaml:39:16 - Unknown word (NOCACHE)
./sdk/communication/azure-communication-phonenumbers/test/recordings/test_phone_number_administration_client.test_list_purchased_phone_numbers_from_managed_identity.yaml:426:9 - Unknown word (Wlkp)
./sdk/communication/azure-communication-phonenumbers/test/recordings/test_phone_number_administration_client.test_list_purchased_phone_numbers_from_managed_identity.yaml:432:16 - Unknown word (NOCACHE)
./sdk/communication/azure-communication-phonenumbers/test/recordings/test_phone_number_administration_client.test_list_purchased_phone_numbers.yaml:430:9 - Unknown word (uncx)
./sdk/communication/azure-communication-phonenumbers/test/recordings/test_phone_number_administration_client.test_list_purchased_phone_numbers.yaml:430:24 - Unknown word (Vxlh)
./sdk/communication/azure-communication-phonenumbers/test/recordings/test_phone_number_administration_client.test_list_purchased_phone_numbers.yaml:436:16 - Unknown word (NOCACHE)
./sdk/communication/azure-communication-phonenumbers/test/recordings/test_phone_number_administration_client.test_purchase_phone_number_from_managed_identity.yaml:33:13 - Unknown word (Muyiv)
./sdk/communication/azure-communication-phonenumbers/test/recordings/test_phone_number_administration_client.test_purchase_phone_number_from_managed_identity.yaml:75:25 - Unknown word (Ekvl)
./sdk/communication/azure-communication-phonenumbers/test/recordings/test_phone_number_administration_client.test_purchase_phone_number_from_managed_identity.yaml:114:15 - Unknown word (YSAECH)
./sdk/communication/azure-communication-phonenumbers/test/recordings/test_phone_number_administration_client.test_purchase_phone_number_from_managed_identity.yaml:155:18 - Unknown word (Umph)
./sdk/communication/azure-communication-phonenumbers/test/recordings/test_phone_number_administration_client.test_purchase_phone_number_from_managed_identity.yaml:195:13 - Unknown word (Fwuq)
./sdk/communication/azure-communication-phonenumbers/test/recordings/test_phone_number_administration_client.test_purchase_phone_number_from_managed_identity.yaml:195:21 - Unknown word (Xrnxvm)
./sdk/communication/azure-communication-phonenumbers/test/recordings/test_phone_number_administration_client.test_purchase_phone_number_from_managed_identity.yaml:234:23 - Unknown word (Nowlenu)
./sdk/communication/azure-communication-phonenumbers/test/recordings/test_phone_number_administration_client.test_purchase_phone_numbers.yaml:37:12 - Unknown word (Zcjpu)
./sdk/communication/azure-communication-phonenumbers/test/recordings/test_phone_number_administration_client.test_purchase_phone_numbers.yaml:37:26 - Unknown word (bocpg)
./sdk/communication/azure-communication-phonenumbers/test/recordings/test_phone_number_administration_client.test_purchase_phone_numbers.yaml:126:26 - Unknown word (Kdbp)
./sdk/communication/azure-communication-phonenumbers/test/recordings/test_phone_number_administration_client.test_purchase_phone_numbers.yaml:171:25 - Unknown word (Qbjk)
./sdk/communication/azure-communication-phonenumbers/test/recordings/test_phone_number_administration_client.test_search_available_phone_numbers_from_managed_identity.yaml:33:17 - Unknown word (Lkmu)
./sdk/communication/azure-communication-phonenumbers/test/recordings/test_phone_number_administration_client.test_search_available_phone_numbers_from_managed_identity.yaml:43:16 - Unknown word (NOCACHE)
./sdk/communication/azure-communication-phonenumbers/test/recordings/test_phone_number_administration_client.test_search_available_phone_numbers_from_managed_identity.yaml:84:16 - Unknown word (NOCACHE)
./sdk/communication/azure-communication-phonenumbers/test/recordings/test_phone_number_administration_client.test_search_available_phone_numbers_from_managed_identity.yaml:125:16 - Unknown word (NOCACHE)
./sdk/communication/azure-communication-phonenumbers/test/recordings/test_phone_number_administration_client.test_search_available_phone_numbers_with_invalid_country_code.yaml:42:16 - Unknown word (NOCACHE)
./sdk/communication/azure-communication-phonenumbers/test/recordings/test_phone_number_administration_client.test_search_available_phone_numbers.yaml:37:13 - Unknown word (TPAY)
./sdk/communication/azure-communication-phonenumbers/test/recordings/test_phone_number_administration_client.test_search_available_phone_numbers.yaml:47:16 - Unknown word (NOCACHE)
./sdk/communication/azure-communication-phonenumbers/test/recordings/test_phone_number_administration_client.test_search_available_phone_numbers.yaml:86:9 - Unknown word (Chua)
./sdk/communication/azure-communication-phonenumbers/test/recordings/test_phone_number_administration_client.test_search_available_phone_numbers.yaml:86:27 - Unknown word (Zetg)
./sdk/communication/azure-communication-phonenumbers/test/recordings/test_phone_number_administration_client.test_search_available_phone_numbers.yaml:92:16 - Unknown word (NOCACHE)
./sdk/communication/azure-communication-phonenumbers/test/recordings/test_phone_number_administration_client.test_search_available_phone_numbers.yaml:131:24 - Unknown word (Misqs)
./sdk/communication/azure-communication-phonenumbers/test/recordings/test_phone_number_administration_client.test_search_available_phone_numbers.yaml:137:16 - Unknown word (NOCACHE)
./sdk/communication/azure-communication-phonenumbers/test/recordings/test_phone_number_administration_client.test_update_phone_number_capabilities_from_managed_identity.yaml:35:16 - Unknown word (NOCACHE)
./sdk/communication/azure-communication-phonenumbers/test/recordings/test_phone_number_administration_client.test_update_phone_number_capabilities_from_managed_identity.yaml:84:16 - Unknown word (NOCACHE)
./sdk/communication/azure-communication-phonenumbers/test/recordings/test_phone_number_administration_client.test_update_phone_number_capabilities_from_managed_identity.yaml:120:9 - Unknown word (ltowox)
./sdk/communication/azure-communication-phonenumbers/test/recordings/test_phone_number_administration_client.test_update_phone_number_capabilities_from_managed_identity.yaml:126:16 - Unknown word (NOCACHE)
./sdk/communication/azure-communication-phonenumbers/test/recordings/test_phone_number_administration_client.test_update_phone_number_capabilities_from_managed_identity.yaml:166:16 - Unknown word (NOCACHE)
./sdk/communication/azure-communication-phonenumbers/test/recordings/test_phone_number_administration_client.test_update_phone_number_capabilities_with_invalid_phone_number.yaml:35:10 - Unknown word (Hkqjgmti)
./sdk/communication/azure-communication-phonenumbers/test/recordings/test_phone_number_administration_client.test_update_phone_number_capabilities_with_invalid_phone_number.yaml:41:16 - Unknown word (NOCACHE)
./sdk/communication/azure-communication-phonenumbers/test/recordings/test_phone_number_administration_client.test_update_phone_number_capabilities.yaml:33:24 - Unknown word (Ftsw)
./sdk/communication/azure-communication-phonenumbers/test/recordings/test_phone_number_administration_client.test_update_phone_number_capabilities.yaml:39:16 - Unknown word (NOCACHE)
./sdk/communication/azure-communication-phonenumbers/test/recordings/test_phone_number_administration_client.test_update_phone_number_capabilities.yaml:92:16 - Unknown word (NOCACHE)
./sdk/communication/azure-communication-phonenumbers/test/recordings/test_phone_number_administration_client.test_update_phone_number_capabilities.yaml:138:16 - Unknown word (NOCACHE)
./sdk/communication/azure-communication-phonenumbers/test/recordings/test_phone_number_administration_client.test_update_phone_number_capabilities.yaml:176:17 - Unknown word (Bkiso)
./sdk/communication/azure-communication-phonenumbers/test/recordings/test_phone_number_administration_client.test_update_phone_number_capabilities.yaml:182:16 - Unknown word (NOCACHE)
```

**Comment by petrsvihlik** [comment]
Note: the files in the `_shared` folder are owned by the ACS Auth team and will be fixed as part of #22636 for all modalities at once

**Comment by petrsvihlik** [comment]
Once fixed, we should remove the package from the spellchecker whitelist - #22199
